src/core/connection.c

```diff
@@ -1844,6 +1849,13 @@ QuicConnStart(
     UdpConfig.OwningProcess = Configuration->OwningProcess;
 #endif
 
+    UdpConfig.UseQTIP = Connection->Settings.QTIPEnabled;
+    QuicTraceLogConnInfo(
+        IndicateQTIPSettingsBubbledDown,
+        Connection,
+        "Bubbling down UseQTIP setting: %hhu",
+        UdpConfig.UseQTIP);
```

Comment:
This log should be unnecessary, IMO, because QuicConnApplyNewSettings would log the value of every settings.

Reply:
Fixed.